core/src/main/java/org/apache/iceberg/util/TableScanUtil.java

```diff
@@ -140,7 +142,7 @@ public static <T extends PartitionScanTask> List<ScanTaskGroup<T>> planTaskGroup
       long splitSize,
       int lookback,
       long openFileCost,
-      Types.StructType projectedPartitionType) {
+      Types.StructType groupingKeyType) {
```

Comment:
Initially, we did not have much clarify on how we wanted to call it. Renaming for consistency with other places.